docs/swaggerv3.yaml

```diff
@@ -0,0 +1,1036 @@
+openapi: 3.0.1
+info:
+  title: checkout-backend-merchant-v3.API
+  description: 'For details, see the [Checkout API Guide](https://vippsas.github.io/vipps-developer-docs/docs/APIs/checkout-api).'
+  version: v3
```

Comment:
Is the v3 API ready to be published?

Reply:
That we will have to discuss. Think Roar is working on it as well